code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -61,11 +59,48 @@
 	maxHealth = 50
 	health = 50
 	sight = SEE_MOBS|SEE_OBJS|SEE_TURFS
-	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash/long,/obj/effect/proc_holder/spell/targeted/telepathy/eldritch)
+	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash/long,/obj/effect/proc_holder/spell/pointed/manse_link,/obj/effect/proc_holder/spell/pointed/trigger/blind/eldritch)
```

Comment:
```suggestion
	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash/long,/obj/effect/proc_holder/spell/pointed/manse_link,/obj/effect/proc_holder/spell/targeted/telepathy/eldritch,/obj/effect/proc_holder/spell/pointed/trigger/blind/eldritch)
```
let them keep normal telepathy so that they can still spook and prank people from behind maint walls to pass the time

speaking of which, have you actually checked if spellcasting (for all of these spell targeting methods) works through walls (provided that you have x-ray)?

Reply:
maybe set the range of eldritch telepathy to 10 while you're at it

actually, could you use that vision range selecting spell from the knowledge devil boon to allow raw prophets to choose whichever vision range works for them? then you could bump these ranges up to 11 again